src/arfsdao.ts

```diff
@@ -408,15 +616,6 @@ export class ArFSDAO {
 			// Get the drives transaction ID
 			drive.txId = node.id;
 		});
-
```

Comment:
I think the diff got weird because of function ordering... but is there code missing here for private drives?

Reply:
Grabbing the Parent Folder ID is missing here. I think when we try to create a private folder with only a drive ID, that transaction will be missing the "Parent-Folder-Id" tag

We will need this from the public drive getter:

```ts

		const txData = await this.arweave.transactions.getData(drive.txId, { decode: true });
		const dataString = await Utf8ArrayToStr(txData);
		const dataJSON = await JSON.parse(dataString);

		// Get the drive name and root folder id
		drive.name = dataJSON.name;
		drive.rootFolderId = dataJSON.rootFolderId;
		```